wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -816,13 +835,40 @@ leverage the API capabilities.
     readFees             = decodeUtf8 $ encodePretty $ genExample @(WalletResponse EstimatedFees)
     readNodeInfo         = decodeUtf8 $ encodePretty $ genExample @(WalletResponse NodeInfo)
     readTransactions     = decodeUtf8 $ encodePretty $ genExample @(WalletResponse [Transaction])
-
+    readUtxoStatistics   = decodeUtf8 $ encodePretty $ genExample @(WalletResponse UtxoStatistics)
 
 -- | Provide an alternative UI (ReDoc) for rendering Swagger documentation.
 swaggerSchemaUIServer
     :: (Server api ~ Handler Swagger)
     => Swagger -> Server (SwaggerSchemaUI' dir api)
-swaggerSchemaUIServer = redocSchemaUIServer
+swaggerSchemaUIServer =
+    swaggerSchemaUIServerImpl redocIndexTemplate redocFiles
+    where
+        redocIndexTemplate :: Text
+        redocIndexTemplate = [text|
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>ReDoc</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+      body { margin: 0; padding: 0; }
+    </style>
+    <script>
+        // Force Strict-URL Routing for assets relative paths
+        (function onload() {
+            if (!window.location.href.endsWith("/")) {
+                window.location.href += "/";
+            }
+        }());
+    </script>
+  </head>
+  <body>
+    <redoc spec-url="../SERVANT_SWAGGER_UI_SCHEMA"></redoc>
+    <script src="redoc.min.js"> </script>
+  </body>
+</html>|]
```

Comment:
TODO: This can be removed, already part of develop. I'll update the base branch to include the change :+1:

Reply:
done